admin/installer/cloudformation.py

```diff
@@ -72,332 +136,275 @@ def _sibling_lines(filename):
         return f.readlines()
 
 
-def _get_cluster_size():
+def print_flocker_docker_template(options):
```

Comment:
I don't think this should take an argparse option object. Instead, it should take just the options it needs, and the caller can do the processing needed to get the argparse options out.